test/integration/smart-loan.test.ts

```diff
@@ -302,10 +285,10 @@ describe('Smart loan', () => {
       await priceProvider.connect(oracle).setPrice(toBytes32('USD'), estimatedAVAXPriceFor1USDToken);
 
 
-      await loan.invest(toBytes32('USD'), toWei("1500", usdTokenDecimalPlaces));
```

Comment:
The reason for this change is that we can't (as it's not supported in hardhat yet) pin a specific block number to fork for and over the time the price of the USD token changes so sometimes it results in the loan to still be solvent even though we expect it not to be.